solutions/0000-0099/0026-Remove-Duplicates-from-Sorted-Array-easy.md

```diff
@@ -0,0 +1,111 @@
+---
+description: 'Author: @pankajsirari222| https://leetcode.com/problems/remove-duplicates-from-sorted-array/'
+tags: [Array, Two Pointers]
+---
+
+# 0026-Remove-Duplicates-from-Sorted-Array-easy
+
+## Problem Link
+
+https://leetcode.com/problems/remove-duplicates-from-sorted-array/
+
+## Problem Statement
+
+Given an integer array `nums` sorted in non-decreasing order, **remove the duplicates** in-place such that each unique element appears only **once**.
+The **relative order** of the elements should be kept the **same**.
+
+Since it is impossible to change the length of the array in some languages, you must instead have the result be placed in the **first part** of the array `nums`. 
```

Comment:
instead of `nums`, use $nums$

Don't use backtick, use LaTex for any literals references.

Change line 18, 19, 21 & 23 as well.